src/mapInteractor.js

```diff
@@ -90,38 +90,54 @@ var mapInteractor = function (args) {
       actions: [{
         action: geo_action.pan,
         input: 'left',
-        modifiers: {shift: false, ctrl: false}
+        modifiers: {shift: false, ctrl: false},
+        owner: 'geo.mapInteractor',
```

Comment:
We don't.  It is a convenience for other applications that may want to modify the default actions.  They can ask for actions based on name and owner to be certain that it is the default action and not one that was created somewhere else.